docs/troubleshooting.md

```diff
@@ -24,3 +24,4 @@ discusses potential solutions to the same.
 | ls: reading directory `<mountpath>/<path>`:  Input/output error                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Find out if you have any object(s) with name/prefix having `//` in it or starting with `/`, in the mounted GCS bucket (use `gsutil ls gs://<bucket>/<path>` to find out). If yes, move/rename such objects to name/prefix not having `//` or starting with `/` (e.g. for object/prefix `A//B`, use `gsutil -m mv -r gs://<bucket>/A//* gs://<bucket>/A/`), or delete it (use `gsutil -m rm -r A//`). Refer [semantics](semantics.md#unsupported-object-names) for more details.                                                                                                                                        |
+| Experiencing hang while executing `ls` on a directory containing large number of files/directories.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Happens when ls triggers `stat` for each list entries other than listing. Fix (a) Use `ls --color=never` or `\ls` to remove the stat part of `ls`. Or (b) Increase the metadata cache ttl/capacity to serve the stat faster from cache (from the first execution as cache is populate as part of listing only). This is a generic solution and work for listing where stat is required. E.g. `ls -lah`                                                                                                                                                                                                                 |
```

Comment:
be explicit, that stat will be served from cache if ls is made on the same directory multiple times.

Reply:
Added a note -`ls` will be faster from the first execution.